src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -32,6 +32,29 @@
   display: flex;
 }
 
+.rightArea {
+  grid-area: right-area;
+}
+
+.colorBar {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 95%;
+}
+
+.colorBarAxis {
+  display: block;
+  overflow: visible;
+  width: min-content;
+}
+
+.gradient {
+  border: solid black 1px;
```

Comment:
The border around the gradient is needed to my opinion but it shifts the ticks by the border size. The ticks are placed according to the height from `useMeasure` that includes the border. I don't know how to fix it yet

Reply:
You can try to use `box-sizing: content-box`. If you then set `gradientHeight` on the `svg` element, it should become 2px shorter than the colour bar. You can then use `align-self: center` on the axis to align it vertically.

If all of this doesn't work, use `padding-top: 1px` on the axis.